testkit/src/main/scala/ch/epfl/scala/bsp/testkit/client/TestClient.scala

```diff
@@ -26,6 +27,8 @@ class TestClient(
   private implicit val executionContext: ExecutionContextExecutor =
     ExecutionContext.fromExecutor(Executors.newCachedThreadPool())
 
+  private implicit val gson: Gson = new Gson()
```

Comment:
I think that `gson` does not need to be implicit. All the method using it are private, so there is no way to pass a different `Gson` instance.

```suggestion
  private val gson: Gson = new Gson()
```